### PR TITLE
Javascript javascript3 week2

### PR DESCRIPTION
**Pull request by hparul**

### CODE REVIEW

File: javascript/javascript3/week2/task_1.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  return new Promise((Resolve) => {
+    setTimeout(() => {
+      Resolve(`is  called asynchronously after ${resolveAfter} seconds`);
+    }, resolveAfter*1000);
+  });
+}
+callMeAsynchronously(5)
+  .then((returnedObject) => {
+    console.log(`the function  ${returnedObject}`);
+  })
```
Comment (florinmaticu) [inline javascript/javascript3/week2/task_1.js:26]: Missing semicolon here.

File: javascript/javascript3/week2/task_1.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    console.log(error);
+  });
+
+//api
+//const apiPromise=new Promise((resolve,reject=>{
+
+function toFetchData() {
+  return new Promise((Resolve) => {
+    setTimeout(() => {
+      Resolve();
```
Comment (florinmaticu) [inline javascript/javascript3/week2/task_1.js:73]: Small error here. Function `Resolve` has one argument.
`Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)
lib.es2015.promise.d.ts(33, 34): An argument for 'value' was not provided.`

**Review by florinmaticu — CHANGES_REQUESTED** [review]
Good job...especially with the `move-elements`. Some small remarks I have added only

File: javascript/javascript3/week2/task_1.js (hunk 1 above)
Comment (hparul) [reply javascript/javascript3/week2/task_1.js]: done

File: javascript/javascript3/week2/task_1.js (hunk 2 above)
Comment (hparul) [reply javascript/javascript3/week2/task_1.js]: done

**Pull request merged**